FlowCal/io.py

```diff
@@ -1243,36 +1258,68 @@ def resolution(self, channels=None):
         else:
             return self._resolution[channels]
 
-    def hist_bins(self, channels=None, nbins=None, scale='linear'):
+    def hist_bins(self, channels=None, nbins=None, scale='linear', **kwargs):
```

Comment:
should the `scale` default be `logicle` I was under the impression we want to make that the new default everywhere (Excel UI, plotting function defaults, here, etc.)?